Documentation/commands.md

```diff
@@ -317,6 +316,42 @@ Work in progress. Please contribute!
 
 Work in progress. Please contribute!
 
+### Logging
+
+By default, rkt will send logs directly to stdout/stderr, allowing them to be caputered by the invoking process.
+On host systems running systemd, rkt will attempt to integrate with journald on the host.
```

Comment:
I suggest to add:

Journald integration requires `systemd` version xxx on the host and `systemd` version yyy in stage1.